analysis/Sentiment_Analysis.py

```diff
@@ -1,18 +1,22 @@
-from analysis.analysis import Analysis
+from analysis.Analysis import Analysis
 import os
-import tensorflow.keras
-# TODO this can be removed once the word_index is saved and loaded in
-from tensorflow.keras.datasets import imdb
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2' # suppresses tf info and warning logs
```

Comment:
I had to change the way we were importing some of this to make the cmd deliverable work